code/mathematical-algorithms/russian_peasant_multiplication/russian_peasant.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace russian_peasant
+{
+    class russ_peasant
+    {
+        int a, b;
+        public russ_peasant(int number1,int number2)
+        {
+            a = number1;
+            b = number2;
+        }
+        public int multiply()
```

Comment:
Instead of having to make a new object for every new pair of numbers, consider making a static method called multiply that takes parameters a and b instead, and remove the private variables a and b along with the constructor.

Reply:
making function would be better then 😕

Reply:
Not necessarily. It just seems extra to have to create a new object every time.

Reply:
@arnavb  ohk bro i am changing to static then 😄